rtps-r52/shell-tests.c

```diff
@@ -204,3 +204,32 @@ rtems_shell_cmd_t shell_cmd_test_link_shmem_trch = {
     NULL, NULL,                                /* alias, next */
     0, 0, 0                                    /* mode, uid, gid */
 };
+
+static int shell_test_mbox_rtps_hpps(int argc RTEMS_UNUSED,
+                                         char *argv[] RTEMS_UNUSED)
+{
+    if(argc <= 1){
```

Comment:
Formatting: here and everywhere else, please follow convention in rest of the code and keep a space between if/else/while/etc, conditionals in `()`, and braces.  E.g., `if (argc <= 1) {`